doc/commands.md

```diff
@@ -0,0 +1,100 @@
+# Documentation for LEDpp-AVR
+
+## Warning
+
+This documentation describes the developing purpose, not the cuzrrent status, so some functions may not be implemented yet.
+
+## Command structure
+
+| Startbyte |  commmand   |    parameters        |  Endbyte   |
+|-----------|-------------|----------------------|------------|
+| char: 'S' | char: a-z   | byte / int / float   | char: 'E'  |
+
+
+## commands for entire LED Array
+
+### Single color:
+
+All LEDs have the same color
+
+command: 'a'
+
+values: byte r, byte g, byte b
+
+example: `a 255 255 255` for white color
+
+
+
+### Blink:
+
+All LEDs have the same color, blinking with periodical with an off-time
+
+command: 'b'
+
+values: byte r, byte g, byte g, byte b, byte periode, float off-time (0-1)
+
+example: `b 255 0 0 20 0.5` for blinking in red color periodical every 20 time units with 50% off-time
```

Comment:
Didn't we decided to use fixed-length ASCII number representations? And probably `50` [%] instead of `0.5`?
Then this should be `b 255 000 000 020 050`.